### DISCUSSION TITLE
Windows 3.0 Works in Standard and Real Mode!

### DISCUSSION THREAD

**Discussion by ghost** [copy/v86#518] [Show and tell]
i tested Windows 3.0 Hard Disk Image , it works fine!
in 386-enhanced mode, mouse is broken but Standard and Real Mode are fine!
 it runs on top of MS-DOS 3.31 and hard disk image can be downloaded from here:
https://mega.nz/file/V94QRS6Q#FY1qiCqd6VovolSJH0p8A58wGQ7o90jTSzdAcYyPYWY

**Comment by ReconEvacuateDEVX** [comment]
neato
